src/plugins/orangehrmBuzzPlugin/config/routes.yaml

```
@@ -43,6 +43,13 @@ apiv2_buzz_comment:
   requirements:
     shareId: '\d+'
 
+apiv2_buzz_url_validation:
+  path: /api/v2/buzz/shares/url
```

```suggestion
apiv2_buzz_validation_links:
  path: /api/v2/buzz/validation/links
```
better to use consistent naming